AwsCryptographicMaterialProviders/dafny/AwsCryptographyKeyStore/src/Index.dfy

```diff
@@ -123,7 +123,7 @@ module {:extern "software.amazon.cryptography.keystore.internaldafny"}
       ddbClient := ddbClient
       )
     );
-    return Success(client as IKeyStoreClient);
+    return Success(client);
```

Comment:
Just OoC, was this actually necessary, or just cleanup?

Reply:
I think this was necessary.
But if I am honest, I was throwing ideas at the wall...
I needed to make this work.